packages/ra-no-code/assets/ra-orders-1.csv

```diff
@@ -1,11 +1,11 @@
 id,reference,date,customer_id,basket.product_id,basket.quantity,total_ex_taxes,delivery_fees,tax_rate,taxes,total,status,returned
-408,5TFGTA,2021-04-23T02:07:38.273Z,17,87,1,108.28,5.38,0.2,22.73,136.39,ordered,false
```

Comment:
The customer_ids weren't pointing to actual records in the customers csv, which caused warnings on import